tensorboard/webapp/metrics/views/right_pane/settings_view_component.ng.html

```diff
@@ -74,6 +74,22 @@ <h3 class="section-title">Scalars</h3>
       >Ignore outliers in chart scaling</mat-checkbox
     >
   </div>
+
+  <div class="control-row scalars-partition-x">
+    <mat-checkbox
+      [checked]="scalarPartitionX"
+      (change)="scalarPartitionXToggled.emit()"
+      >Partition non-monotonic X axis</mat-checkbox
```

Comment:
Hm, I'm struggling to come up with better phrasing, but I'm wondering if we might be able to get away with slightly a longer visible label that explains how the partition will occur, without always having to read the full tooltip. Maybe something like:

"Partition non-monotonic time series into multiple monotonic series"
?

Reply:
I will keep the terse phrasing at least for now.